qrenderdoc/Windows/Dialogs/CaptureDialog.ui

```diff
@@ -666,6 +741,9 @@ Click here to set up Vulkan capture.</string>
       </item>
       <item>
        <widget class="QPushButton" name="launch">
+        <property name="toolTip">
+         <string>Trigger a capture of the selected program</string>
```

Comment:
This isn't your fault as you've just copied it but I see now that this tooltip is pretty misleading/confusing. I think this might be better:

> Launch the selected program with RenderDoc enabled for capture.

You don't have to update the C# UI version, just fix it here.